app/forms/member_form.rb

```diff
@@ -0,0 +1,40 @@
+class MemberForm
+  include ActiveModel::Model
+  include Virtus
+
+  attribute :email, String
+  attribute :board_id, Integer
+
+  attr_reader :member_form
+
+ validates :email, presence: true, format: { with: /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/, message: 'incorrect email'}
+ validates :board_id, presence: true
+ validate :user?
+
+  def save
+    if valid?
+      persist!
+      true
+    else
+      false
+    end
+  end
+
+  private
+  def user?
+    user = User.find_by(email: email)
+    board = Board.find(board_id)
+    unless user
```

Comment:
and maybe you can do this in different way ( without else )
```
errors.add(:email, 'There is not such user') and return unless user
errors.add(:email, 'You are owner of this board') if user.id == board.owner_id
```
I also consider that this should be check in separate validations
```
validate :user_presence
validate :user_owner if: :find_user

def user_presence
   errors.add(:email, 'There is not such user') unless find_user
end

def user_owner?
  errors.add(:email, 'You are owner of this board') if user.id == board.owner_id
end

def find_user
  return @user if defined?(@user)
  @user = User.find_by(object.user_id)
end
```